specification/eventgrid/data-plane/Microsoft.Communication/stable/2018-01-01/AzureCommunicationServices.json

```diff
@@ -7,6 +7,54 @@
   },
   "paths": {},
   "definitions": {
+    "AcsIncomingCallEventData": {
+      "description": "Schema of the Data property of an EventGridEvent for an Microsoft.Communication.IncomingCall event",
+      "type": "object",
+      "properties": {
+        "to": {
```

Comment:
The naming is not consistent with other properties that use the CommunicationIdentifierModel.

Reply:
i dont understand

Reply:
Other properties using this type are suffixed with CommunicationIdentifier, e.g. https://github.com/Azure/azure-rest-api-specs/blob/a4ad2d15d8086c162d48ef1c8cdb9118d8128ed8/specification/eventgrid/data-plane/Microsoft.Communication/stable/2018-01-01/AzureCommunicationServices.json#L350

Reply:
they use CommunicationIdentifierModel 
![image](https://user-images.githubusercontent.com/65305780/235796288-5c22c360-7dcb-42ff-a988-8e6bd3e477c1.png)

Reply:
Yes, the comment is about the name of the property. Should it be suffixed with CommunicationIdentifier? Also, see the other naming comments left in the Api View about these properties.

Reply:
ooh I see. I believe the goal was to treat it for easy as possible to just be incoming call from and incoming call to. But i will follow up with the call team to get clarification